src/Education/Commands/RuleHelpUserControl.xaml

```diff
@@ -2,20 +2,25 @@
              xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
              xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
              xmlns:vsshell="clr-namespace:Microsoft.VisualStudio.Shell;assembly=Microsoft.VisualStudio.Shell.15.0"
-             xmlns:vsTheming="clr-namespace:Microsoft.VisualStudio.PlatformUI;assembly=Microsoft.VisualStudio.Shell.15.0"
+             xmlns:vsTheming="clr-namespace:Microsoft.VisualStudio.PlatformUI;assembly=Microsoft.VisualStudio.Shell.15.0"             
              >
-
     <UserControl.Resources>
         <!-- Defines the standard set of style used by the document elements -->
-        
-        <!--<Style TargetType="Paragraph">
-            <Setter Property="Margin" Value="3" />
-        </Style>-->
         <Style TargetType="FlowDocument">
             <Setter Property="Background" Value="{DynamicResource {x:Static vsshell:VsBrushes.WindowKey}}"/>
             <Setter Property="Foreground" Value="{DynamicResource {x:Static vsshell:VsBrushes.WindowTextKey}}"/>
             <Setter Property="FontSize" Value="{DynamicResource {x:Static vsshell:VsFonts.EnvironmentFontSizeKey}}"/>
             <Setter Property="FontFamily" Value="{DynamicResource {x:Static vsshell:VsFonts.EnvironmentFontFamilyKey}}"/>
+
+            <!-- Optimise whitespace and word-breaks for readability -->
+            <Setter Property="IsOptimalParagraphEnabled" Value="True" />
+            <Setter Property="IsHyphenationEnabled" Value="True" />
+
+            <!-- Reduce blurriness of font outlines. -->
+            <!-- Note: same settings as for the Help About box. Also, changes to these properties when
+                 debugging are not picked up immediately - you need to recompile to see the difference. -->
+            <Setter Property="TextOptions.TextRenderingMode" Value="ClearType" />
+            <Setter Property="TextOptions.TextFormattingMode" Value="Ideal" />
```

Comment:
@bigfluffycookie thanks for the info about which `TextOptions` to use.
Setting the `TextOptions` does make a difference, but you have to recompile to see it (unlike other properties, which you can change on the fly when debugging and see the difference immediately).

#### Without
![image](https://user-images.githubusercontent.com/10757559/217203673-9cc36014-897b-4061-b9cf-e8ab778540e3.png)


#### With
![image](https://user-images.githubusercontent.com/10757559/217203744-0b946993-f353-4982-9387-cc39c1b79846.png)